kratos/containers/array_1d.h

```diff
@@ -106,15 +106,6 @@ class	array_1d	: public boost::numeric::ublas::vector_expression< array_1d<T, N>
         //sin esto no funciona en windows!!
         //std::fill (data().begin(), data().end(), value_type	());
     }
-    explicit BOOST_UBLAS_INLINE
-    array_1d (size_type array_size):
-        vector_expression<self_type> ()
-    {
-        //sin esto no funciona en windows!!
-        //std::fill (data().begin(), data().end(), value_type	());
```

Comment:
maybe we can at least bring back this version

and also throw an error in case `array_size` is larger than `N`

Reply:
I am not sure throwing an error is a good idea in release. The overhead of having the possibility would be high here.

For adding as debug error, I have my doubt if the overhead of exception handling mechanism is ommited in the release where the if is false or not.